### PR TITLE
Feature to introduce Dev Containers to Conformitron

### PR DESCRIPTION
**Pull request by elamaran11**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by elamaran11** [comment]
@mikemcd3912 Lets merge this when you get a chance. Feel free to try out if you have time. This is going to be useful for our customers.

File: README.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ Refer the example [here](https://github.com/aws-samples/eks-anywhere-addons/blob
 
 ## Pre-requisite (Linux/MacOS)
 
+Please use the dev container configuration in the `.devcontainer` folder with [devpod](devpod.sh) or any other dev container environment to create minikube cluster with all required pre-requisites such helm, kubectl and flux for local testing. You can skip over to local testing section if you use dev container environment.
```
Comment (shapirov103) [inline README.md:63]: with all *the* required prerequisites

such *as* helm

skip over to *the* local testing section

**Review by shapirov103 — COMMENTED** [review]
Minor grammar

File: README.md (hunk 1 above)
Comment (elamaran11) [reply README.md]: Fixed. Pls check

**Comment by elamaran11** [comment]
@shapirov103 Please merge, typo fixed.

**Review by shapirov103 — APPROVED** [review]
LGTM

**Pull request merged**